pkg/wakaama/include/lwm2m_client_connection.h

```diff
@@ -5,10 +5,10 @@
 * are made available under the terms of the Eclipse Public License v1.0
 * and Eclipse Distribution License v1.0 which accompany this distribution.
 *
-* The Eclipse Public License is available at
+* The Eclipse Public License is available at:
 *    http://www.eclipse.org/legal/epl-v10.html
-* The Eclipse Distribution License is available at
-*    http://www.eclipse.org/org/documents/edl-v10.php.
+* The Eclipse Distribution License is available at:
+*    http://www.eclipse.org/org/documents/edl-v10.php
```

Comment:
Not needed

Reply:
It was to satisfy the tool. It's a simple shell script that is not sophisticated enough to divide between a dot in a URL and a dot at the end of line :worried:

Reply:
I can revert it.

Reply:
?